EnglishFilter/sections/specific.txt

```diff
@@ -858,6 +858,8 @@ imgbox.com##div[id^="aad-header-"]
 dr-farfar.net##.footerBar > iframe
 ||mrfog.com^$domain=dr-farfar.net
 leechall.com##.download-main-content > a[href^="https://bit.ly/"][target="_blank"]
+vpnmentor.com##.main-content-holder > div.article-thanks-wrap
```

Comment:
One second, please move this rule to the AdGuard Social filter -> `specific.txt`

Reply:
If you can't do this - please write here. I'll make the changes directly.